### DISCUSSION TITLE
Beginner Tutorial/Quest Guide

### DISCUSSION THREAD

**Discussion by Crealotte** [sunflower-land/sunflower-land#2029] [Ideas]
With the public release of the game, I feel new players are a bit left in the dark, needing to look for answers. I know this because I was one of them ... It gets a bit frustrating that a game that might seem easy does not explain itself.

I came up with the idea to write a tutorial/quest guide for newer players or even for experienced players but for newer features.

I will share here my idea for the 5 first tutorial quests and a list of what I already have included or in mind.

Please share any ideas you have that need to be included in a Quest Guide.

Ideas are already taken into account:
- Helios
- Tools
- Grubshop
- Fire pit and Kitchen
- Sync
- Skills
- Expansions

already thinking about:
- goblins retreat
- mints
- withdrawing
- fruits
- mushrooms
- refreshing minds of players on recipes for new items

- quests that are optional between long times of leveling/expanding progress.

my first 5 quests I will show in here to give an idea of what I have been working on:

Welcome to your very own sunflower island! You have made it through the war and your barrel got you all the way to these sunny shores.
-----------------------------------
You must be hungry! If you help me with my beloved sunflowers I’ll teach you how to cook.
[Quests Sunflower Land (1).docx](https://github.com/sunflower-land/sunflower-land/files/10475576/Quests.Sunflower.Land.1.docx)

“Continue”
-----------------------------------
Do you see those sunflowers? Can you harvest them for me?

Head over to the market <image of market> to sell the sunflowers, you will need the SFL to buy more seeds! 

O harvest 100 sunflowers

“Continue”
-----------------------------------
Well, thanks for helping me with that! Now you have enough SFL to get some potatoes and prepare to get your strength up!

O harvest 20 potato seeds

Now go to the fire pit <image>

O cook 2 mashed potatoes

“Continue”
-----------------------------------
Mashed potatoes are the easiest comfort food that exists on the islands!
To eat the food you just cooked, click your little bumpkin next to the mailbox. <image mailbox>

“Continue”
-----------------------------------
You seem stronger already! Eating delicious homemade food is the way to get XP which will level up your strength and give you access to new buildings and skills, which we will learn about in a bit. First, let’s see how much more you can grow!

O cook 18 mashed potatoes

“Continue”
-----------------------------------

**Comment by future2back** [comment]
I really think something like that would be a priority for game development. Also communication with the players is really bad. Announcements and news high priority.

**Comment by Ogrekin** [comment]
Good initiative to start a thread like this... Will try to give my input and knowledge from a more experienced player's point of view!!!!

**Comment by Galphath** [comment]
I like how you are weaving the story into the tutorial!

**Comment by Polysemouse** [comment]
We really need a tutorial for newcomers to the game!  The values in your tutorial example might need some tweaks, but it does a good job on guiding new players

**Comment by adamhannigan** [comment]
@Crealotte Great initiative! This is something we have planned over the next month to work on - we will continue to collaborate with you 🤗 

Typically games provide small rewards to encourage early positive feedback (let the player know they are on the right track). In the case of these mini quests, do you think we should provide additional rewards (e.g. a small amount of SFL)

**Reply by Crealotte** [reply]
I'll keep writing them for you to read :) I would love some rewards, either in SFL or some resources to get them going up a bit quicker.
